### PR TITLE
Update OWNERS

### PR DESCRIPTION
**Pull request by cardelearht**
Please add cardelea  reviewers to list

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -6,6 +6,7 @@ reviewers:
   - jianzhangbjz
   - tomasdavidorg
   - LuboTerifaj
+  - cardelea
```
Comment (LuboTerifaj) [inline OWNERS:9]: Is this correct? Shouldn't it be `cardelearht`?

**Review by LuboTerifaj — COMMENTED** [review]
Hello @cardelearht , could you please update your commit message? "Please add cardelea  reviewers list" is not ideal as a part of commit message.

File: OWNERS (hunk 1 above)
Comment (cardelearht) [reply OWNERS:9]: @LuboTerifaj   it shod be cardelearht my bad

**Comment by LuboTerifaj** [comment]
@cardelearht , could you please rebase your branch with master? Thank you!

**Pull request closed without merging**